public/app/features/alerting/unified/components/rules/RuleDetailsActionButtons.tsx

```diff
@@ -172,7 +167,7 @@ export const RuleDetailsActionButtons = ({ rule, rulesSource, isViewMode }: Prop
             variant="primary"
             icon="apps"
             target="_blank"
-            href={`d/${encodeURIComponent(dashboardUID)}?viewPanel=${encodeURIComponent(panelId)}`}
+            href={`d/${encodeURIComponent(dashboardUID)}?viewPanel=${encodeURIComponent(panelId)}${scenes ? '&scenes' : ''}`}
```

Comment:
do we really need these two url changes? is the feature toggle not enough?

Reply:
Yeah, after implementing this I also felt like it was unnecessarily messy. I'll revert them.